src/Sulu/Bundle/AdminBundle/Resources/js/containers/SmartContent/stores/smartContentConfigStore.js

```diff
@@ -30,6 +30,9 @@ class SmartContentConfigStore {
             sortBy: config.sorting.length > 0 ? config.sorting[0].name : undefined,
             sortMethod: config.sorting.length > 0 ? 'asc' : undefined,
             tagOperator: config.tags ? 'or' : undefined,
+            types: config.types && config.types.length > 0 ? config.types.map((type)=>{
```

Comment:
Why do we need to check for the existince of `config.types`, but not for `config.sorting`? And the `map` could be written easier:

```javascript
config.types.map((type) => type.value)
```

And if we need to break the line, I would do that at the `?` and `:` of the ternary operator, IMO it reads easier that way:

```javascript
config.types.length > 0
    ? config.types.map((type) => type.value)
    : undeifned
```